cmd/jobsResume.go

```diff
@@ -286,7 +286,8 @@ func (rca resumeCmdArgs) process() error {
 		destinationSAS: rca.DestinationSAS,
 	}); err != nil {
 		return err
-	} else if credentialInfo.CredentialType == common.ECredentialType.OAuthToken() {
+	} else if credentialInfo.CredentialType == common.ECredentialType.OAuthToken() ||
+		credentialInfo.CredentialType == common.ECredentialType.Anonymous() {
```

Comment:
Why Anonymous?

Reply:
I think this smells a bit fishy as well... SAS tokens are anonymous, and get upset when you combine them with OAuth.

Reply:
@nakulkar-msft did you by chance mean AutoLogin here?

Reply:
Yes. I blame IDE :(